src/core/sys/darwin/mach/thread_act.d

```diff
@@ -135,3 +140,103 @@ version (i386)
     kern_return_t thread_resume(thread_act_t);
     kern_return_t thread_get_state(thread_act_t, thread_state_flavor_t, thread_state_t*, mach_msg_type_number_t*);
 }
+else version (AppleARM)
```

Comment:
Could you include a comment linking to the original definitions?

Reply:
Done.